packages/protocol/engine/tsc/tscmanager.go

```diff
@@ -83,28 +62,18 @@ func (o *TSCManager) AddUnconfirmedBlock(block *booker.Block) {
 func (o *TSCManager) orphanBeforeTSC(minAllowedTime time.Time) {
```

Comment:
Maybe extend the comment: `// orphanBeforeTSC removes all elements with key time earlier than the given time. If a block is not accepted by this time, it becomes orphaned.`